tests/webapp/api/test_perfcompare_api.py

```diff
@@ -224,6 +224,8 @@ def test_perfcompare_results_with_only_one_run_and_diff_repo(
 
     base_perf_data_values = [32.4]
     new_perf_data_values = [40.2]
+    base_perf_data_replicates = [20]
```

Comment:
nit: It could be better to have a test with multiple values in the replicates since it'll almost always have more than 1.